pkg/platform/runtime/runtime.go

```diff
@@ -57,7 +57,7 @@ func newRuntime(target setup.Targeter, an analytics.Dispatcher, svcModel *model.
 	}
 
 	if !rt.store.MarkerIsValid(target.CommitUUID()) {
-		if target.OnlyUseCache() {
+		if target.ReadOnly() && target.InstallFromDir() == nil {
```

Comment:
Why does we need to assert InstallFromDir here? The behaviour of "this runtime is not set up, please update it" should still work the same way for offline installs too.

Reply:
Oops, this was leftover from a previous attempt.